cyprecice/cyprecice.pyx

```diff
@@ -242,41 +280,44 @@ cdef class Interface:
         """
         return self.thisptr.requiresInitialData ()
 
-    def requires_reading_checkpoint (self):
+
+    def requires_writing_checkpoint (self):
```

Comment:
I guess you swapped the order of these two functions to make it consistent with `precice/src/precice/Participant.hpp`? Makes the diff a bit messy, but makes sense. Maybe consider moving this minor change as an individual commit directly to develop to improve readability here.